learing_journal/tests.py

```diff
@@ -137,15 +151,15 @@ def test_create_error(new_session):
     Test if an error msg shows up on the page
     in case an empty entry on the new_entry page is submitted.
     """
-    from .views.default import home
-    result = home(dummy_http_request_post('', '', '', new_session))
-    assert result['error_msg'] == "Can't submit empry entry"
+    from .views.default import create
+    result = create(dummy_http_request_post('', '', '', new_session))
+    assert result['error'] == "You are missing the Title or the Body"
```

Comment:
You also need to check that your update view changes models as you expect it to.